src/state/slices/accountSpecifiersSlice/selectors.ts

```diff
@@ -8,3 +9,16 @@ export const selectAccountSpecifierStrings = (state: ReduxState) =>
   state.accountSpecifiers.accountSpecifiers.map(accountSpecifier =>
     Object.entries(accountSpecifier)[0].join(':')
   )
+
+// returns a CAIP2-indexed object with all the `caip2:pubkeyish` accounts for that chainId
+export const selectPubkeyishByChainId = (state: ReduxState, chainId: CAIP2) =>
```

Comment:
It should be also renamed in this PR https://github.com/shapeshift/web/pull/1357#discussion_r841317133

Reply:
Updated to match the other PR